source/lib/projectz.js

```diff
@@ -386,8 +386,9 @@ class Projectz {
 
 		// Extract repository information
 		/* eslint no-magic-numbers: 0 */
-		let repo = this.mergedPackageData.repository.url || null
-		const githubMatch = (this.mergedPackageData.repository.url || this.mergedPackageData.homepage).match(/github\.com\/(.+?)(?:\.git|\/)?$/)
+		let repo = this.mergedPackageData.repository || null
+		const githubMatch = (this.mergedPackageData.repository.url || this.mergedPackageData.homepage).match(/github\.com\/(.+?)(?:\.git|\/)?$/) ||
+			(this.mergedPackageData.repository.match(/^(.+\/.+)$/) && !this.mergedPackageData.repository.match(/^(.+:.+\/.+)$/))
```

Comment:
Actually, I think I understand what you mean now. Leave it with me. Will merge it in and adjust.